source/versions/filenames.rst

```diff
@@ -0,0 +1,121 @@
+describe the information provided.
+
+SpongeAPI
+=========
+
+The format of the filename is:
+
+``spongeapi-<APIMajor>.<LatestAPIMinorRelease>-shaded.jar``
+
+**OR**
+
+``spongeapi-<APIMajor>.<LatestAPIMinorRelease>.<Date>.<Time>-<BuildNumber>-shaded.jar``
+
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``APIMajor``         | The major version of SpongeAPI implemented by this file (the ``X`` in                         |
+|                      | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``LatestAPI``        | The minor version of SpongeAPI implemented by this file (the ``Y`` in                         |
+| ``MinorRelease``     | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``Date``             | The date when the build job ran.                                                              |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``Time``             | The time when the build job ran.                                                              |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``<BuildNumber>``    | The build number of Sponge. Supply this number when reporting bugs or seeking support.        |
++----------------------+-----------------------------------------------------------------------------------------------+
+
+.. note::
+
+    The first format without the build information is the release and recommended build format. The second format with 
+    build information is the *SNAPSHOT* format. This version is still in development.
+    
+    See our 
+    `build system 
+    <https://docs.spongepowered.org/stable/en/plugin/buildsystem.html#creating-a-plugin-without-a-build-system>`_ page 
+    for an explanation of the ``-shaded`` label.
+
+.. _sponge-forge-file-name:
+
+SpongeForge
+===========
+
+.. note::
+
+    Sponge changed our versioning policy in October 2018. **Beginning with** SpongeAPI 8.0, the following information 
+    applies. See :doc:`legacy-versions` for information on older versions.
+
+When you download SpongeForge, the name of the file includes a Forge build number which this version of SpongeForge is 
+compatible with. Other builds, even ones differing by only a few build numbers, are not officially supported.
+
+However, SpongeForge usually updates to a new Forge build fairly soon after it's released, so you needn't worry about 
+always having to run an outdated Forge version in order to use SpongeForge.
```

Comment:
I'm not sure this is really true, or useful (reflecting phit's comments above). When a Forge release breaks SpongeForge, we try to get an update out, but otherwise it's a bit ad-hoc. Normally updating Forge is a safe operation, but there is no particular rule I can think of that covers it nicely. Other than "try it and see what happens".

Reply:
I used phit's comment to rewrite those paragraphs. We may need to make more changes after you review the new update.

Reply:
I realized you were mentioning the paragraph after phit's comment. So, I re-worked the changes to handle both comments. We still may need to make further changes, but I think you will like the new changes.